web/client/reducers/layers.js

```diff
@@ -372,6 +374,24 @@ function layers(state = { flat: [] }, action) {
             settings: settings
         });
     }
+
+    case SHOW_LAYER_SWIPE_SETTINGS: {
+        const swipeSettings = assign({}, state.swipeSettings, {
```

Comment:
Also this should be in a swipe reducer, separated from the map layers.

rename all things (actions, actionCreators and state variables) this way:

`reducers/swipe.js`:
swipeSettings changes to the to `active: true| false`

`actions/swipe.js`:
action can be only one SET_ACTIVE = "SWIPE:SET_ACTIVE", with one parameter, `active`
actionCreators in  1 or 2 (What you prefer): or `setActive(active)` or `activate() `and `deactivate()`.

don't forget the epic.